### DISCUSSION TITLE
Solucan lastik sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/BilsemPython#87] [Soru Cevap]
Elimizde 10 cm boyunda olan ve sonsuza dek gerip uzatabileceğimiz özdeş bir lastik ip var. Bu ipin A ucunu bir duvara bağlıyoruz. B ucunu da bir arabaya bağlıyoruz. A ucuna bir solucan yerleştiriyoruz. Solucan ipin B ucuna doğru gidiyor. Saniyede 1 cm yol alıyor. Solucan 1 saniye süreyle gidince duruyor. Solucan durunca araba lastik ipi çekerek B ucunu 10 cm ileriye götürüyor ve duruyor. Araba durunca solucan yine 1 saniye süreyle gidiyor ve duruyor. Solucan durunca araba lastik ipin B ucunu yine 10 cm ileriye götürüyor ve duruyor. Ve bu olay hiç durmadan böyle tekrarlanıyor. Solucan lastik ipin B ucunu yakalayabilir mi? Yakalarsa ne zaman? Programlayınız.

**Comment by CinarSatir** [comment]
```python
a = 10
s = 0
sa=0
while s<a:
    s = s+1
    a = a+10
    y = a/(a-10)
    s = s*y
    sa=sa+1
print(sa)
```

**Comment by tolga0102** [comment]
```python
can=0
a=10
sn=1
while can<=a:
    can+=1
    a+=10
    can*=a/(a-10)
    sn+=1
print(sn)

```

**Comment by OmerB2** [comment]
```python
s = 0
so = 0
a = 10
while so<a:
    so= so+1
    a = a+10
    y = a/(a-10)
    so  = so*y
    s=s+1
print(s)
```

**Comment by Berkek007** [comment]
```
solucan=0
araba=10
s=0

while solucan<araba:
    solucan+=1
    araba+=10
    lastik=(araba/(araba-10))
    solucan*=lastik
    s+=1

print(s)
```

**Comment by slos8888** [comment]
```
#solucan=a
#araba=b
#lastik=l
a=0
b=10
x=0

while a<b:
    a+=1
    b+=10
    l=(b/(b-10))
    a*=l
    x+=1

print(x)

```
